packages/airnode-deployer/src/utils/logger.ts

```diff
@@ -84,3 +90,21 @@ export function debugMode(mode: boolean) {
 export function inDebugMode() {
   return debugModeFlag;
 }
+
+export function setSecret(secret: string) {
+  secrets.push(secret);
+}
+
+export function replaceSecrets(input: string, secrets: string[]) {
+  let output = input;
+  secrets.forEach((secret) => (output = output.replace(secret, '*'.repeat(secret.length))));
+
+  return output;
+}
+
+export function setLogsDirectory(path: string) {
+  logsDirectory = path.endsWith('/') ? path.slice(0, -1) : path;
```

Comment:
You don't need to "sanitize" the directory path if you then use `path.join` to create a file path (which we should be doing anyway).

```ts
fs.appendFileSync(path.join(logsDirectory, `deployer-${logFileTimestamp}.log`), `${timestamp}: ${sanitizedLogs}\n`);
```